app/browser/api/ledger.js

```diff
@@ -442,11 +442,15 @@ const pruneSynopsis = (state) => {
     return state
   }
 
-  appActions.onPruneSynopsis(json.publishers)
+  module.exports.onPruneSynopsisAction(state, json.publishers)
```

Comment:
why was appAction removed?

Reply:
We're mocking out/recoding the appAction calls so that test flow doesn't get lost in the reducer.